### PR TITLE
[Develop] new component Badge

### PR DESCRIPTION
**Pull request by naimdanialjohar**
# Description

new component Badge. [Jira ticket](https://kfit-asia.atlassian.net/browse/FAVE-17112)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings

### CODE REVIEW

File: src/Avatar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
```
Comment (AladdinNullptr) [inline src/Avatar/index.tsx:1]: Why does the badge branch contains avatar file changes?

Comment (naimdanialjohar) [reply src/Avatar/index.tsx:1]: the badge component uses on avatar component for the gray circle

Comment (AladdinNullptr) [reply src/Avatar/index.tsx:1]: You should branch from develop in this case, the branch should ideally contain badge related changes to minimize conflicts and have cleaner branch tree

**Review by AladdinNullptr — APPROVED** [review]

File: src/Badge/index.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { Badge as AntBadge, BadgeProps as AntBadgeProps } from 'antd';
+import './style.css';
```
Comment (alex-safian) [inline src/Badge/index.tsx:3]: Please add a line break here

File: src/Badge/index.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+export type BadgeProps = AntBadgeProps;
+
+const Badge: React.FC<BadgeProps> = props => {
+  return <AntBadge {...props} />;
+};
+
+Badge.defaultProps = {};
```
Comment (alex-safian) [inline src/Badge/index.tsx:11]: Remove this line, if the component doesn't have any default props.

**Pull request merged**